### ISSUE TITLE
Track by geographic location

### ISSUE THREAD

**Issue by oluoluoxenfree** [RoboPython/transcode#6]
Would be cool to be able to drill down and see who is saying what where!

**Comment by aychedee** [comment]
What data would we need to store in order to implement this?

**Comment by JessFairbairn** [comment]
Were you thinking of a map display @oluoluoxenfree ? If so I'd imagine coordinates would be the most useful.

**Comment by oluoluoxenfree** [comment]
A map would be cool!

**Comment by aychedee** [comment]
Okay, so if we have co-ordinates we store them, and if we have generic country info we turn that into co-ordinates and store that?

**Comment by oluoluoxenfree** [comment]
Yeah sounds good to me! Even if we just start off with a list of place names I think it'd be cool.
